proactive-jupyter-kernel/kernel.py

```diff
@@ -1431,6 +1434,62 @@ def __set_generic_information_from_input_data__(self, proactive_common, input_da
             for gen_info in input_data['generic_info']:
                 proactive_common.addGenericInformation(gen_info[0], gen_info[1])
 
+    def __set_job_nodesource__(self, proactive_common, input_data):
+        if 'nodesource' in input_data:
+            self.__kernel_print_ok_message__('Adding node source information ...\n')
+            NODE_SOURCE_NAME = input_data['nodesource']
+            proactive_common.addGenericInformation("NODESOURCENAME", NODE_SOURCE_NAME)
```

Comment:
Is this necessary? Generic information + selection script to set a target NS?

Reply:
Yes.